packages/project/src/shared/tree-utils/set-instance-style.ts

```diff
@@ -22,7 +22,11 @@ export const setInstanceStyleMutable = (
   }
 
   for (const update of updates) {
-    cssRule.style[update.property] = update.value;
+    if (update.value.type === "unset") {
+      delete cssRule.style[update.property];
+    } else {
+      cssRule.style[update.property] = update.value;
+    }
```

Comment:
if we keep early return coding style, in this case I would write  a `continue`
```suggestion
    if (update.value.type === "unset") {
      delete cssRule.style[update.property];
      continue;
    }
    cssRule.style[update.property] = update.value;
```

Reply:
I honestly don't like to put it in every condition. Early return works well when we have only return or continue alone. Though here is symmetrical logic I would expect to see on the same depth.

Reply:
You might be right about it, idk, I was just trying to make it all consistent and basically stop using `else`. I guess I am trying to avoid else all together because I got burned by complex if/else statements.

In this particular case either way its readable, so up to you, unless we ban  `else` by linter haha